src/methods.js

```diff
@@ -24,7 +24,9 @@ var methods = {};
             verdictLevel,
             score;
 
-        if (options === undefined) { return; }
+        if (typeof(options) === "undefined") {
```

Comment:
JSLint is finding these issues:
src/methods.js:27:19	Expected exactly one space between 'typeof' and '('.
src/methods.js:27:13	Unexpected 'typeof'. Use '===' to compare directly with undefined.

Could you change this line to `if (options === undefined) {` ?